.github/workflows/release.yml

```
@@ -1,25 +1,21 @@
 name: 🚀Release
-run-name: 🚀${{ inputs.release-type }} Release ${{ inputs.dry-run == true && '(Dry Run)' || '' }}
+run-name: 🚀${{ inputs.release_type }} Release ${{ inputs.dry-run == true && '(Dry Run)' || '' }}
 
 
 defaults:
   run:
     shell: pwsh
 
 
-env:
-  runs-on: ubuntu-latest
-
-
 on:
   workflow_dispatch:
     inputs:
-      release-type:
+      release_type:
```

Rename workflow input:

```suggestion
      release-type:
```